product_by_supplier/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Product by supplier info
+========================
+
+This module categorizes each product item by supplier. It allows for users to be able to view a compiled list of products supplied by the supplier.
+This information is the corresponding to the supplier list on each product.
+
+Users can also directly add new products to the supplier's list.
```

Comment:
Can you mention here that the module adds a new menu item that allows the user to manage their supplier/product info?